File: PRACTICA/ORDSUBARR/ORDSUBARR.cs
using System;
using System.Linq;
class OrdenarSubarreglos {
    static void Main() {
        int[][] matriz = {
            new int[]{3,1,2},
            new int[]{9,5,4},
            new int[]{7,8,6}
        };
        int[][] asc = matriz.Select(f => f.OrderBy(x=>x).ToArray()).ToArray();
        int[][] desc = matriz.Select(f => f.OrderByDescending(x=>x).ToArray()).ToArray();
        int[][] alt = matriz.Select((f,i)=> (i%2==0 ? f.OrderBy(x=>x) : f.OrderByDescending(x=>x)).ToArray()).ToArray();
        
        Console.WriteLine("Ascendente:");
        foreach(var fila in asc) Console.WriteLine(string.Join(" ", fila));
    }
}

File: PRACTICA/ORDSUBARR/ORDSUBARR.py
matriz = [[3,1,2],[9,5,4],[7,8,6]]

asc = [sorted(fila) for fila in matriz]
desc = [sorted(fila, reverse=True) for fila in matriz]
alt = [sorted(fila, reverse=(i%2==1)) for i,fila in enumerate(matriz)]

print("Ascendente:", asc)
print("Descendente:", desc)
print("Alternado:", alt)
